File: Programs/WeChat/考试查询（微信小程序）/微信/pages/ZKScoreResult/ZKScoreResult.js
const app = getApp();

//中考成绩
import ZkScore from '../Template/ZkScore.js'
//考生信息对象
import ExamineeInfo from '../Template/ExamineeInfo.js'

Page({
  data: {
    ChineseScore:0,
    MathematicsScore:0,
    EnglishScore:0,
    HistoryScore:0,
    GeographyScore:0,
    BiologyScore:0,
    ChemistryScore:0,
    PhysicsScore:0,
    PoliticsScore:0,
    SumScore:0
  },
  onLoad:function(){
    this.setData({
      ChineseScore: ZkScore.ChineseScore,
      MathematicsScore: ZkScore.MathematicsScore,
      EnglishScore: ZkScore.EnglishScore,
      HistoryScore: ZkScore.HistoryScore,
      GeographyScore: ZkScore.GeographyScore,
      BiologyScore: ZkScore.BiologyScore,
      ChemistryScore: ZkScore.ChemistryScore,
      PhysicsScore: ZkScore.PhysicsScore,
      PoliticsScore: ZkScore.PoliticsScore,
      SumScore: ZkScore.SumScore
    })
  },
  ContinueQuery: function () {
    wx.navigateTo({
      url: '../ZKQuery/ZKQuery',
    })
  },
  scoreQuery: function () {
    ExamineeInfo.data.QueryType = 4
    wx.request({
      url: 'http://localhost:9001/Home/Query',
      type: "GET",
      data: ExamineeInfo.data,
      success: function (data) {
        if (data.data.Existence) {
          wx.navigateTo({
            url: '../ZKAdmissionsResults/ZKAdmissionsResults?SchoolName=' + data.data.SchoolName,
          })
        }
        else {
          wx.navigateTo({
            url: '../NoQuery/NoQuery',
          })
        }
      }
    })
  }

})

File: Programs/WeChat/考试查询（微信小程序）/广东考试查询/ExamQuery/ExamQuery/Models/ZkScoreResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ExamQuery.Models
{
    public class ZkScoreResult : BaseModel
    {
        public string StudentName { get; set; }
        /// <summary>
        /// 语文成绩
        /// </summary>
        public float ChineseScore { get; set; }
        /// <summary>
        /// 数学成绩
        /// </summary>
        public float MathematicsScore { get; set; }
        /// <summary>
        /// 英语成绩
        /// </summary>
        public float EnglishScore { get; set; }
        /// <summary>
        /// 历史成绩
        /// </summary>
        public float HistoryScore { get; set; }
        /// <summary>
        /// 地理成绩
        /// </summary>
        public float GeographyScore { get; set; }
        /// <summary>
        /// 生物成绩
        /// </summary>
        public float BiologyScore { get; set; }
        /// <summary>
        /// 化学成绩
        /// </summary>
        public float ChemistryScore { get; set; }
        /// <summary>
        /// 物理成绩
        /// </summary>
        public float PhysicsScore { get; set; }
        /// <summary>
        /// 政治成绩
        /// </summary>
        public float PoliticsScore { get; set; }
        /// <summary>
        /// 总分
        /// </summary>
        public float SumScore { get; set; }
    }
}